helpers/cron-requester/README.md

```diff
@@ -0,0 +1,68 @@
+ <div align="center">
+ <!-- <img align="center" width="180" src="https://i.imgur.com/62VsVXD.png" /> -->
+  <h2>Band Data Requester</h2>
+  <blockquote>A light-weight node.js tool to query data on BandChain periadically</blockquote>
+</div>
+
+## ⭐️ Features
+
+- Support cron-style scheduling
+- Support configuration via `json` file
+- No-installation required with [`npx`](https://www.npmjs.com/package/npx)
+
+## 📦 Prerequisite
+
+You need to create a `config.json` file in your machine.
+
+See example [`config.json`](./config.js) for requesting data from GuanYu [devnet](https://guanyu-devnet.cosmoscan.io/) every 5 minutes:
```

Comment:
Rephrase to "See config.json file (also shown below) for an example of requesting data from Band's GuanYu devnet every 5 minutes". And the file link for `config.json` should be `./config.json` not `./config.js`?